### PR TITLE
updata Adamw.py

### PR DESCRIPTION
**Pull request by hua-zi**
### PR types

Others

### PR changes

Docs

### Description

英文文档修复 

一处：\*\*Notes\*\*:   ->  Notes:
两处：out.backward()  -> loss.backward()

### CODE REVIEW

**Review by sunzhongkai588 — DISMISSED** [review]
LGTM

**Comment by sunzhongkai588** [comment]
这篇文的还有个typo，也提个pr修一下吧

- note格式错了，应该写成     `Note: ` ，参考[【英文模板】](https://www.paddlepaddle.org.cn/documentation/docs/zh/dev_guides/api_contributing_guides/api_docs_guidelines_cn.html#yingwenmuban)

image
image

**Comment by hua-zi** [comment]
好的，已经在这个pr中修改

**Review by sunzhongkai588 — APPROVED** [review]
LGTM
